Rakefile

```diff
@@ -176,16 +203,26 @@ namespace :git do
       sh "git add ."
       sh "git ci -m 'Update version to #{version}'"
     end
-    force_update(branch, nil)
+    force_update(branch, nil, false)
   end
 
-  { :status => nil, :push => nil, :reset => '--hard', :diff => nil }.each do |command, options|
+  { :show => nil, :status => nil, :reset => '--hard', :diff => nil }.each do |command, options|
     desc "git #{command} on all the repos"
     task command => :clone do
       run_command "git #{command} #{options}".strip
     end
   end
 
+  desc 'git push on all the repos'
```

Comment:
Nice!

Reply:
I have this script, works for all subdirectories:

```bash
#!/bin/bash

all() {
  (
    cd $2
    cd ..
    echo $(bash -c "$1" 1>/dev/null 2>&1 && echo "[+]" || echo "[ ]") $(pwd)
  )
}
export -f all

echo "$*" for all repos
export COMMAND="$*"
find . -maxdepth 2 -type d -name '.git' | xargs -P 10 -L1 -I{} bash -c 'all "$COMMAND" "{}"'
```

```
~/source/rspec-dev/repos $ git checkout drop-old-rubies
git co drop-old-rubies for all repos
[ ] /Users/pirj/source/rspec-dev/repos/rspec-collection_matchers
[ ] /Users/pirj/source/rspec-dev/repos/rspec.github.io
[+] /Users/pirj/source/rspec-dev/repos/rspec
[+] /Users/pirj/source/rspec-dev/repos/rspec-mocks
[+] /Users/pirj/source/rspec-dev/repos/rspec-support
[+] /Users/pirj/source/rspec-dev/repos/rspec-expectations
[ ] /Users/pirj/source/rspec-dev/repos/rspec-rails
[+] /Users/pirj/source/rspec-dev/repos/rspec-core
```

If you push it somewhere on the `PATH`, it's usable for other projects if you happen to work with dissected monoliths.